inactiveLink.html

```diff
@@ -0,0 +1,21 @@
+<head>
```

Comment:
Not a fan of this file, at least in its current form.

If we have a bunch of future link targets we should establish standard link target names for them, and in the meantime the links should take you to an entry in a list of descriptions of what each link is supposed to point at.  I'm not sure what the best way to engineer our site to do that would be.

Can we not stick to markdown instead of raw HTML?